egs/demask/README.md

```diff
@@ -0,0 +1,219 @@
+<div align="center">
+[Inspiration](#inspiration) •
+[What it does](#what-it-does) •
+[How we built it](#how-we-built-it) •
+[Asteroid highlights](#asteroid-highlights) •
+[Challenges and Accomplishments](#challenges-we-ran-into) •
+[Acknowledgement](#acknowledgement)
+</div>
+
+## Inspiration
+
+#### Asteroid's inspiration
+It all started during a speech processing research project.
+We wanted to go fast and tried "ready-to-use" speech separation/speech enhancement models.
+We quickly realized that nothing was really working as expected,
+and spent our time fixing other people's bugs instead of doing research.
+This is the struggle that inspired Asteroid, and motivated us to open-source our
+code with things that _just work_.
+
+While sharing research code is already a step in the right direction,
+sharing readable and reproducible code should be the standard.
+Asteroid aims at empowering developers and researchers with tools that makes this even easier.
+
+#### DeMask's inspiration
+
+We were having a fast call and one of us was in the train.
+We couldn't hear him well because he was wearing a mask.
+We directly thought about building surgically masked speech enhancement model with
+Asteroid and we were on it !
+
+## What it does
+
+#### About Asteroid
+
+Asteroid is an audio source separation toolkit built with PyTorch and PyTorch-Lightning.
+Inspired by the most successful neural source separation systems, it provides all neural
+building  blocks  required  to  build  such  a  system.   To  improve  reproducibility,
+recipes  on  common  audio source  separation datasets  are provided,
+including all the steps from data download/preparation through training to evaluation.
+
+Asteroid exposes all levels of granularity to the user from simple layers to complete ready-to-use models.
+Our pretrained models are hosted on the [asteroid-models community in Zenodo](https://zenodo.org/communities/asteroid-models).
+Loading pretrained models is trivial and sharing them is also made easy with asteroid's CLI.
+
+You can check, [our landing page](https://asteroid-team.github.io/),
+[our repo](https://github.com/mpariente/asteroid),
+[our latest docs](https://mpariente.github.io/asteroid/)
+and [our model hub](https://zenodo.org/communities/asteroid-models).
+
+To try Asteroid, install it with `pip install asteroid` !
+
+#### About DeMask
+
+DeMask is a simple, yet effective, end-to-end muffled speech enhancement built with Asteroid.
```

Comment:
i would remove "muffle" and just say "enhancement for speech when wearing a face mask"

Reply:
Ok